src/fswAlgorithms/attGuidance/flybyPoint/_UnitTest/test_flybyPoint.py

```diff
@@ -0,0 +1,285 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+#
+#   Unit Test Script
+#   Module Name:        flybyPoint
+#   Author:             Riccardo Calaon
+#   Creation Date:      May 08, 2023
+#
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+from Basilisk import __path__
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import flybyPoint
+from Basilisk.utilities import SimulationBaseClass, macros, unitTestSupport
+from Basilisk.utilities import RigidBodyKinematics as rbk
+
+bskPath = __path__[0]
+fileName = os.path.basename(os.path.splitext(__file__)[0])
+
+
+@pytest.mark.parametrize("initPos", [[-5e7, 7.5e6, 5e5]])  # m - r_CN_N
+@pytest.mark.parametrize("initVel", [[2e4, 0, 0]])  # m/s - v_CN_N
+@pytest.mark.parametrize("dTsim", [10, 100])  # s
+@pytest.mark.parametrize("dTfilter", [0, 60, 600])  # s
+@pytest.mark.parametrize("accuracy", [1e-12])
+
```

Comment:
:large_orange_diamond: This line can be removed.

Reply:
Done